libraries/joomla/document/jsonapi.php

```diff
@@ -41,6 +41,7 @@ public function __construct($options = array())
 
 		// Set mime type to JSON-API
 		$this->_mime = 'application/vnd.api+json';
+		$this->_type = 'jsonapi';
```

Comment:
For first instancese we can use fixed type but for future would be awesome to have some switch for differents json types